numpy/core/tests/test_simd.py

```diff
@@ -156,6 +156,79 @@ def test_tobits(self):
             tobits = bin(self.tobits(vdata))
             assert tobits == bin(data_bits)
 
+    def test_andc(self):
+        if self.sfx not in ("b8"):
+            return
+        andc_simd = getattr(self.npyv, f"andc_b8")
+        # create the vectors
+        data = self._data()
+        rdata = self._data(reverse=True)
+        vdata = self._load_b(data)
+        vrdata = self._load_b(rdata)
+        # check andc
+        sandc = [(~x & y) & 0xFF for x, y in zip(rdata, data)]
+        vandc = andc_simd(vrdata, vdata)
+        assert sandc == vandc
+
+    def test_orc(self):
+        if self.sfx not in ("b8"):
+            return
+        orc_simd = getattr(self.npyv, f"orc_b8")
+        # create the vectors
+        data = self._data()
+        rdata = self._data(reverse=True)
+        vdata = self._load_b(data)
+        vrdata = self._load_b(rdata)
+        # check orc
+        sorc = [(~x | y) & 0xFF for x, y in zip(rdata, data)]
+        vorc = orc_simd(vrdata, vdata)
+        assert sorc == vorc
+
+    def test_xnor(self):
+        if self.sfx not in ("b8"):
+            return
+        xnor_simd = getattr(self.npyv, f"xnor_b8")
+        # create the vectors
+        data = self._data()
+        rdata = self._data(reverse=True)
+        vdata = self._load_b(data)
+        vrdata = self._load_b(rdata)
+        # check orc
+        sxnor = [~(x ^ y) & 0xFF for x, y in zip(rdata, data)]
+        vxnor = xnor_simd(vrdata, vdata)
+        assert sxnor == vxnor
```

Comment:
```suggestion
        if self.sfx not in ("b8"):
            return
        # check andc
        sandc = [(~x & y) & 0xFF for x, y in zip(rdata, data)]
        vandc = andc_simd(vrdata, vdata)
        assert sandc == vandc
        # check orc
        sorc = [(~x | y) & 0xFF for x, y in zip(rdata, data)]
        vorc = orc_simd(vrdata, vdata)
        assert sorc == vorc
        # check xnor
        sxnor = [~(x ^ y) & 0xFF for x, y in zip(rdata, data)]
        vxnor = xnor_simd(vrdata, vdata)
        assert sxnor == vxnor
```
you can combine these cases with the above section `test_operators_logical` in order to reduce the amount of code

Reply:
Done! I reduced the code above and also added a test for andc (u8).